src/callbacks/onResize/addPointClick/spaghettiPlot/onResize/addExposure.js

```diff
@@ -0,0 +1,159 @@
+export default function addExposure() {
+    const context = this;
+    this.svg.select('.se-exposure-supergroup').remove();
+
+    //If exposure data exists, annotate exposures beneath x-axis.
+    if (this.edish.exposure.include) {
+        const supergroup = this.svg
+            .insert('g', '.supergroup')
+            .classed('se-exposure-supergroup', true);
+        const dy = 20; // offset from chart
+        const strokeWidth = 5; // width/diameter of marks
+        this.svg.selectAll('.x.axis .tick text').attr('dy', `${dy + strokeWidth * 3}px`); // offset x-axis tick labels
+
+        //top boundary line
+        supergroup.append('line').attr({
+            x1: -this.margin.left,
+            y1: this.plot_height + dy - strokeWidth * 2,
+            x2: this.plot_width,
+            y2: this.plot_height + dy - strokeWidth * 2,
+            stroke: 'black',
+            'stroke-opacity': 0.1
+        });
+
+        //Exposure text
+        supergroup
+            .append('text')
+            .attr({
+                x: -3,
+                y: this.plot_height + dy + strokeWidth,
+                'text-anchor': 'end',
+                textLength: this.margin.left - 3
+            })
+            .text('Exposure');
```

Comment:
On second thought, I think the mouseover information is adequate for now, since there's not a great way to handle labels for people who get multiple treatments.